### PR TITLE
Add pytest-django to the project

### PR DESCRIPTION
**Pull request by shayakrach**
Ran pipenv install --dev pytest-django.
so vagrant up will install pytest.
Add pytest workflow to GitHub.
Created pytest.ini to let pytest know where the project is.
Add dummy test that passes.

### CODE REVIEW

File: .idea/Side-Hustle.iml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yuvalk) [inline .idea/Side-Hustle.iml]: Did you mean to add all these .idea files?
for the very least, they should be their own commit, seems unrelated to this commit
but IMHO, should not be part of project at all

File: Pipfile [hunk 2]
```diff
@@ -5,6 +5,8 @@ verify_ssl = true
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
```
Comment (yuvalk) [inline Pipfile]: I think pytest-django is enough

**Comment by yuvalk** [comment]
what about running the tests?

File: .idea/Side-Hustle.iml (hunk 1 above)
Comment (shayakrach) [reply .idea/Side-Hustle.iml]: I removed them @yuvalk .
They were added by accident in one of the initial PR's.

File: Pipfile (hunk 2 above)
Comment (shayakrach) [reply Pipfile]: Ok, I will remove it.

**Review by assafg95 — APPROVED** [review]

File: Pipfile (hunk 2 above)
Comment (shayakrach) [reply Pipfile]: Done @yuvalk

**Comment by shayakrach** [comment]
Tests are running @yuvalk ... or am I missing something?

**Comment by assafg95** [comment]
@yuvalk can you be more specific on what you mean by running the tests? Are you talking about tests/tests.py file?

File: .idea/Side-Hustle.iml (hunk 1 above)
Comment (yuvalk) [reply .idea/Side-Hustle.iml]: I still see them

**Comment by yuvalk** [comment]
> Tests are running @yuvalk ... or am I missing something?

yeah, that looks fine, sorry

**Review by assafg95 — APPROVED** [review]

**Comment by edenmaor** [comment]
LGTM!

File: .idea/Side-Hustle.iml (hunk 1 above)
Comment (shayakrach) [reply .idea/Side-Hustle.iml]: image

@yuvalk  
Any file in .idea was deleted.

Comment (yuvalk) [reply .idea/Side-Hustle.iml]: navigate to
https://github.com/redhat-beyond/Side-Hustle/pull/19/files
and you'll see there are 14 files.
 many are from the .idea folder
 you need to remove them from the commit.
 not locally
 it is best to add .idea to the project .gitinore file (I think you've done that)

Comment (shayakrach) [reply .idea/Side-Hustle.iml]: @yuvalk 
But the snapshot was taken from https://github.com/redhat-beyond/Side-Hustle/pull/19/files. 

I removed them locally and update that they removed. 
Is there any other solution?

File: .gitignore [hunk 3]
```diff
@@ -138,3 +138,5 @@ nohup.out
 #sqlite3
 db.sqlite3
 
+# Pycharm
+.idea/
```
Comment (EdDev) [inline .gitignore]: This change and the removal of the `.idea/*` files should appear in a different PR. Please extract this out of this change.

File: .github/workflows/pytest.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      container: fedora:32
+      steps:
+         - name: Install Pipenv and Git
+           run: dnf install -y pipenv git
+         - name: Checkout code
+           uses: actions/checkout@v2
+         - name: Setup environment
+           run: pipenv sync --dev
+         - name: Run PyTest
+           run: pipenv run pytest -v
```
Comment (EdDev) [inline .github/workflows/pytest.yml]: I will be better to leave a newline at the end... see the diff warning.

**Review by EdDev — CHANGES_REQUESTED** [review]

**Review by EdDev — APPROVED** [review]
Looks good.

Please make sure to post a following PR to remove those unrelated files that got in by mistake.
